src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,77 @@
+.nav {
+  visibility: visible;
```

Comment:
Why you use this rule. The default value is visible.

Reply:
Fixed. I had forgotten this I don't remember what I was doing.